multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/FileEntryProperties.java

```diff
@@ -0,0 +1,15 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface FileEntryProperties {
+
+    String getGuid();
+
+    String getName();
+
+    String getSpaceGuid();
```

Comment:
Should we consider namespace as additional field?

Reply:
In the other method that was used so far we've never checked the namespace. I think the file id is unique enough